[2023-08-31T03:14:51.999+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509
[2023-08-31T03:14:52.000+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings'
[2023-08-31T03:14:52.034+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509
[2023-08-31T03:14:52.035+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings'
[2023-08-31T03:14:52.066+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.extract_listings manual__2023-08-31T03:14:19.504462+00:00 [queued]>
[2023-08-31T03:14:52.075+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.extract_listings manual__2023-08-31T03:14:19.504462+00:00 [queued]>
[2023-08-31T03:14:52.075+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-08-31T03:14:52.087+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_listings> on 2023-08-31 03:14:19.504462+00:00
[2023-08-31T03:14:52.092+0000] {standard_task_runner.py:57} INFO - Started process 998 to run task
[2023-08-31T03:14:52.095+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'extract_listings', 'manual__2023-08-31T03:14:19.504462+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmpsjbr5lsy']
[2023-08-31T03:14:52.097+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask extract_listings
[2023-08-31T03:14:52.134+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509
[2023-08-31T03:14:52.135+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-08-31T03:14:19.504462+00:00/task_id=extract_listings'
[2023-08-31T03:14:52.137+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.extract_listings manual__2023-08-31T03:14:19.504462+00:00 [running]> on host 1a0e8b05e04d
[2023-08-31T03:14:52.205+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='extract_listings' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T03:14:19.504462+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-31T03:14:19.504462+00:00'
[2023-08-31T03:14:52.206+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: extract_listings() missing 1 required positional argument: 'results'
[2023-08-31T03:14:52.214+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=Autotrader_ETL_DAG, task_id=extract_listings, execution_date=20230831T031419, start_date=20230831T031452, end_date=20230831T031452
[2023-08-31T03:14:52.225+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 97 for task extract_listings (extract_listings() missing 1 required positional argument: 'results'; 998)
[2023-08-31T03:14:52.267+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-31T03:14:52.287+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
